caffe2/utils/SmallVector.h

```diff
@@ -0,0 +1,968 @@
+//===- llvm/ADT/SmallVector.h - 'Normally small' vectors --------*- C++ -*-===//
```

Comment:
Now that we have a unified cmake, we shouldn't copy paste SmallVector again, right? There's already an implementation in ATen